### DISCUSSION TITLE
Welcome to DedSec Discussions!

### DISCUSSION THREAD

**Discussion by dedsec1121fk** [dedsec1121fk/DedSec#1] [Announcements]
Welcome to the GitHub Discussions Page for DedSec!

This space is dedicated to fostering community engagement, facilitating discussions, and sharing ideas related to my project. Whether you’re seeking help, reporting issues, proposing new features, or just want to chat about DedSec this is the place for it.

**Here’s what you can do:**
- **Ask Questions:** Get support from the community or project maintainers.
- **Report Issues:** Let me know about any bugs or problems you encounter.
- **Suggest Features:** Share your ideas for improving the project.
- **Engage in Discussions:** Connect with other users and contributors to discuss DedSec. 

**Community Rules:**
1. **Be Respectful:** Treat all members with kindness and respect. Harassment, discrimination, or offensive behavior I not be tolerated.
2. **No Spam:** Avoid posting irrelevant or promotional content. This space is for constructive dialogue and support.
3. **Be Clear and Concise:** When asking questions or reporting issues, provide as much detail as possible to help others understand and assist effectively.

By participating, you agree to follow these rules to help create a positive and productive environment for everyone. I'm excited to collaborate with you and appreciate your contributions!

**Comment by Nemol8034** [comment]
Hello developers! I don't quite understand how the function works: fox chat
![1761395338390](https://github.com/user-attachments/assets/a3f4a1ab-a1a6-48ae-82d2-0449b8207d92)

**Reply by dedsec1121fk** [reply]
You must put a password (one letter/number/symbol at least or more if you want) and then you must put the same password when you visit the link. :)

**Reply by Nemol8034** [reply]
Thank you!

**Comment by Ir0n2** [comment]
Are the messages in Fox Chat encrypted? I can't find the line where the messages are encrypted or decrypted. Also,  why are the messages sent over cloudflare and not over tor? Tor is free and anonymous. I would've thought an anonymous hacking collective would be more anonymous.

**Reply by dedsec1121fk** [reply]
They're not full encrypted, because for using Tor it would require more time each time to start. I'm trying to make it good for new users that wants to learn.

**Reply by Ir0n2** [reply]
Understandable. Cool project!

**Reply by dedsec1121fk** [reply]
Hey again, I took your ideas and now it has encrypted connection and a new link option for Tor! Thanks again!
